First Last Rules.md

```diff
@@ -0,0 +1,193 @@
+1. First Last is not allowed to ask an Archon to "send him to the backrooms".
+1. First Last is not allowed to introduce himself as Last First.
+1. First Last is not allowed to refer to himself as anything other than "chaotic evil".
+1. First Last is not allowed to run for Republic Parliament.
+1. First Last is not allowed to "assist" Katya Reynolds in "becoming the Empress of Humanity".
+1. First Last is not allowed to install an RBMK reactor on his flagship.
+1. First Last is not allowed to refer to any Southern ship designers as "Nitrogenpocket", "Carbondioxidepocket", "Carbonmonoxidepocket", or "Watervaporpocket".
+1. First Last is not allowed to recreate the Chernobyl incident on his flagship.
+1. First Last is not allowed to hack into the Korath Automaton network and force them to play "BABY SHARK EARRAPE (10 HOUR PERFECT LOOP)".
+1. First Last is not allowed to hack into the Korath Automaton network.
+1. First Last is not allowed to form the "Paid Worlds", "Discount Worlds", "Expensive Worlds", or "Cheap Worlds".
+1. First Last is not allowed to refer to any Southern ship designers as "Methanepocket".
+1. First Last is not allowed to rename the Free Worlds government "the USSR".
+1. First Last is not allowed to argue over whether Katya Reynolds' first name is Katya or Katerina.
+1. First Last is not allowed to express his political beliefs.
+1. First Last is not allowed to express his religious beliefs.
+1. First Last is not allowed to express anything he believes in in any capacity.
+1. First Last is not allowed to refer to Katya as "Hurricane Katyna".
+1. First Last is not allowed to tribute non-human worlds.
+1. First Last is not allowed to tribute monks.
+1. First Last is not allowed to refer to any Southern ship designers using any name involving a gaseous chemical and the word "pocket".
+1. First Last is not allowed to use computers, ships, or any motorized vehicle on Harmony, Girtab.
+1. First Last is not allowed to own a Marauder Fury.
+1. First Last is not allowed to seduce an AI to do his bidding.
+1. First Last is not allowed to own a Marauder Bactrian.
+1. First Last is not allowed to recreate the United States of America within human space.
+1. First Last is not allowed to place any system within jump drive range of the Deep.
+1. First Last is not allowed to seduce any non-human being.
+1. First Last is not allowed to play Fleetwood Mac's "The Chain" at full volume on the bridge of his flagship.
+1. First Last is not allowed to purchase "18 tonnes of cargo, 12 tonnes of outfits, and seven navy cruisers".
+1. First Last is not allowed to say "get that sussy baka!" while referring to anyone on board one of his ships.
+1. First Last is not allowed to name any planet after any female politician born between the years 1900 and 2100.
+1. First Last is not allowed to create a JavaScript framework.
+1. First Last is not allowed to "weaponise the backrooms".
```

Comment:
Original message used American English spelling (weaponize).

Reply:
Sorry, that one slipped by. I did try to normali**z**e on American spellings. 😬